spec/system/token_spec.rb

```diff
@@ -20,7 +20,7 @@
     expect(current_path).to eq(root_path)
   end
 
-  xit 'cannot be reused' do
+  it 'cannot be reused past the drift window' do
```

Comment:
`past the drift window` is irrelevant to the spec here. The spec is testing that specific tokens can't be re-used. Any other detail is an implementation detail.